yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/commons/DockerCreateContainer.java

```diff
@@ -71,11 +71,6 @@
     @CheckForNull
     private String command;
 
-    /**
-     * List variant of #command
-     */
-    private List<String> commands;
```

Comment:
You can't remove fields, because it will lost configuration

Reply:
Ok, will add it again. Did not find any usage, so I removed it.